.github/workflows/deployProd.yml

```diff
@@ -17,6 +17,7 @@ concurrency:
 jobs:
   build_docker:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
Is it not possible to update lines `4-7` with this requirement?

Reply:
Based on our pairing discussion I'll update `deployStg` so that it will do a workflow call to `deployProd` if everything in `deployStg` was successful

Reply:
Dropped that other commit so that this will still be the intended way to require successful conclusion of `deployStg` before the jobs in `deployProd` begin